package.json

```diff
@@ -5,9 +5,10 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
     "test:CI": "CI=true react-scripts test \"-- --coverage\"",
-    "eject": "react-scripts eject"
+    "test": "react-scripts test --watchAll",
+    "eject": "react-scripts eject",
+    "e2e:test": "cypress open"
```

Comment:
Why didn't you use the normal creact-react-app test script?

Reply:
+1 @placideirandora

Reply:
@placide the normal `react-script test` does not do end-to-end testing, it tests the react component. Just to make clear, end-to-end testing means, testing the UI, which the normal testing does not do

Reply:
So, you mean that we will be testing normal endpoints (src) and the UI (public) ?

Reply:
No we will be testing React components (normal testing) and the actual User Interface (not the `public` folder), For example, let's say that when you click a button, you expect it's text to change. We will use `Cypress` to test if the text within the button actually changed after the button click.

Reply:
Indeed! You meant two kinds of testing: react and cypress.